plenum/server/node.py

```diff
@@ -406,6 +394,71 @@ def __init__(self,
         # We need future_primaries to calculate applied primaries correctly
         self.future_primaries_handler = FuturePrimariesBatchHandler(self.db_manager, self)
 
+    def init_req_managers(self):
+        self.write_manager = WriteRequestManager(self.db_manager)
+        self.read_manager = ReadRequestManager()
+        self.action_manager = ActionRequestManager()
+
+    def init_storages(self, storage=None):
+
+        # Config ledger and state init
+        self.db_manager.register_new_database(CONFIG_LEDGER_ID,
+                                              self.init_config_ledger(),
+                                              self.init_config_state())
+
+        # Pool ledger init
+        self.db_manager.register_new_database(POOL_LEDGER_ID,
+                                              self.init_pool_ledger(),
+                                              self.init_pool_state())
+
+        # Domain ledger init
+        self.db_manager.register_new_database(DOMAIN_LEDGER_ID,
+                                              storage or self.init_domain_ledger(),
+                                              self.init_domain_state())
+
+        # Audit ledger init
+        self.db_manager.register_new_database(AUDIT_LEDGER_ID,
+                                              self.init_audit_ledger())
+
+        self.bls_bft = self._create_bls_bft()
```

Comment:
Should we init BLS BFT in `init_storages`? Maybe create a separate method for this?